### DISCUSSION TITLE
谢谢. 问题已经修复好了.

### DISCUSSION THREAD

**Discussion by 13600100046** [getlantern/lantern-forum-cn#91] [Ideas]
既然你们无法保证软件连接稳定，那请问可以把版本退回一个礼拜之前的稳定版吗，请回答我的问题。

**Comment by 13600100046** [comment]
版本7.7.2，坐标中国深圳
现在发现维护者真是哪里漏了堵哪里，之前SG服务器烂，然后集体固定在US服务器，现在US服务器人多了，开始卡了网速超极差，我看你们怎么处理，直接退回一个礼拜前的设置不行吗！！！！！智能选择连接通畅，网速快的服务器有什么问题？？？部分人要升级要新的功能，说实际一点，管我们大部分人什么事？大部分人不就只需要个稳定的梯子吗，你们这样做就是让大多数人为那一小撮人买单，非常不负责任！

**Reply by cqbcqb** [reply]
您好，请问7.7.2是哪里下载的，我在官网下载就是7.7.1，而且一直无法使用

**Reply by 13600100046** [reply]
就是官网下载的，安装完毕会显示已更新到7.7.2

**Comment by MaxMKMax** [comment]
@13600100046 好的。我们会听取您尊贵的反馈。明晚我会把您的建议带到会议上去和团队讨论对策。我们换去SG是因为大部份的用户说US和HK无法使用。没想到SG依然出现问题。我们会尽快解决的。希望您能见谅。就算您是免费用户，我们依然尊重您的意见。

我们都很珍惜每位用户，无论是免费使用者或付费使用者。也包括您。你给的意见对我们来说非常珍贵。希望您留在蓝灯。我们会尽快修复的。谢谢您。

**Reply by 13600100046** [reply]
@Mike777Khoo 我是付费用户来的，只是因为之前SG服务器太不稳定了，所以选择不登录连接US服务器。今天下午网络好多了，登陆了连接的是SG服务器，网络现在非常好，希望保持。

**Comment by MaxMKMax** [comment]
@13600100046 谢谢您美好的反馈。您的支持是我们动力的来源。
